pkg/rtc/participant.go

```diff
@@ -129,6 +129,9 @@ type ParticipantImpl struct {
 	subscribedTo      map[livekit.ParticipantID]struct{}
 	unpublishedTracks []*livekit.TrackInfo
 
+	// stats for signal and data channel
+	dataChannelStats *telemetry.SignalAndDataStats
```

Comment:
Can we make two separate objects here, one for signal and one for data? Suggesting it so that we can keep things clearly separated.

Reply:
It is for data channels only, the signal is in the controller node

Reply:
Got it. I was suggesting something like (after reading the entire PR, left a comment about a factory method also)
```
dataChannelStats *telemetry.BytesTrackStats (or some structure name which does not say `SignalAndDataStats`)
```
and use factory methods to just create either with trackID for data or signal.

Reply:
My thought is that, maybe we can even use your new struct (as it is generic for any trackID) for media tracks also at some point. So, did not want to call it `SignalAndDataStats` and keep it generic.

Reply:
ok, I have the name `BytesTrackStats` first, then sway to the specific `SignalAndDataStats`. Looks like` Bytes` naming win for 2:1